client/components/LearnerViewDirective.js

```diff
@@ -192,26 +205,52 @@ tie.directive('learnerView', [function() {
       </style>
     `,
     controller: [
-      '$scope', 'SolutionHandlerService', 'QuestionDataService',
+      '$scope', '$timeout', 'SolutionHandlerService', 'QuestionDataService',
       'LANGUAGE_PYTHON',
       function(
-          $scope, SolutionHandlerService, QuestionDataService,
+          $scope, $timeout, SolutionHandlerService, QuestionDataService,
           LANGUAGE_PYTHON) {
         var language = LANGUAGE_PYTHON;
-        var question = QuestionDataService.getData();
-        var prompts = question.getPrompts();
-        var currentPromptIndex = 0;
+        var questionSetId = 'strings';
```

Comment:
It is unclear to me how one would make a distinction between something we know will need to be changed that requires a TODO indicator and something we know will need to be changed but does not require a TODO indicator. It may be obvious to the author what is and isn't subtle, but for anyone else who reads this code and would have the same question, the TODO indicates the author is aware of the issue and they intend to change the code when they are able to. I am not convinced that tracking TODOs is an issue.